src/app/notifications/components/SearchOrHighlightFailure/index.tsx

```diff
@@ -27,13 +28,14 @@ export const syncState = (prevState: typeof initialState) => {
 };
 
 // Appears when search searchHighlightManager in PageComponent.tsx fails to handle
-// selecting a search result. It's meant to not be dismissable before shouldAutoDismissAfter elapses
+// selecting a search result or when HighlightScrollTarget can't be find on a page.
```

Comment:
is there anything in here that is actually dependent on the subject being a search result or highlight? we could rename it FlashMessageError and i would feel a little better